chef/cookbooks/nova_dashboard/recipes/server.rb

```diff
@@ -297,8 +277,7 @@
     :memcached_locations => memcached_locations,
     :can_set_mount_point => node["nova_dashboard"]["can_set_mount_point"],
     :can_set_password => node["nova_dashboard"]["can_set_password"],
-    :keystone_api_version => keystone_api_version,
-    :keystone_internal_auth_url => keystone_internal_auth_url,
+    :multi_domain_support => node["nova_dashboard"]["multi_domain_support"],
```

Comment:
I missed that. But I guess I could argue that the user could go back and change keystone to 2.0 afterwards; which would result in the bug. Kind of a edge case, but I'd still protect against it.